src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/PublicationTests.cs

```diff
@@ -0,0 +1,160 @@
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = amendedRelease1Id,
+                        Version = 1,
+                        ReleaseName = "2000",
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        PreviousVersionId = originalReleaseId,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = amendedRelease2Id,
+                        Version = 2,
+                        ReleaseName = "2000",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        PreviousVersionId = amendedRelease1Id,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestPublishedRelease();
+            Assert.Equal(amendedRelease1Id, result.Id);
+        }
+
+        [Fact]
+        public void LatestPublishedRelease_NewDraft()
+        {
+            var release2000Id = Guid.NewGuid();
+            var release2001Id = Guid.NewGuid();
+            var release2002Id = Guid.NewGuid();
+            var publication = new Publication
+            {
+                Releases = new List<Release>
+                {
+                    new()
+                    {
+                        Id = release2000Id,
+                        Version = 0,
+                        ReleaseName = "2000",
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        Published = DateTime.UtcNow.AddDays(-2),
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+
+                    },
+                    new()
+                    {
+                        Id = release2001Id,
+                        Version = 0,
+                        ReleaseName = "2001",
+                        Published = DateTime.UtcNow.AddDays(-1),
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Approved,
+                    },
+                    new()
+                    {
+                        Id = release2002Id,
+                        Version = 0,
+                        ReleaseName = "2002",
+                        Published = null,
+                        TimePeriodCoverage = TimeIdentifier.AcademicYear,
+                        ApprovalStatus = ReleaseApprovalStatus.Draft,
+                    },
+                },
+            };
+
+            var result = publication.LatestPublishedRelease();
+            Assert.Equal(release2001Id, result.Id);
+        }
```

Comment:
I think you could also add tests for LatestPublished_NoPublishedRelease and LatestPublished_NoReleases where we'd expect to catch bugs if we accidently assume at least one Release exists.